app/src/App.js

```diff
@@ -1,35 +1,110 @@
 import * as React from "react";
 
-import { Routes, Route, Link } from "react-router-dom";
-
-import Tasks from "./Tasks";
-
-const App = () => (
-  <>
-    <nav>
-      <Link to="/">Home</Link> | <Link to="dashboard">Dashboard</Link>
-    </nav>
-    <main>
-      <Routes>
-        <Route path="/" element={<Home />} />
-        <Route path="/dashboard" element={<Dashboard />} />
-      </Routes>
-    </main>
-  </>
-);
-
-const Home = () => (
-  <>
-    <h1>{process.env.REACT_APP_TITLE}</h1>
-    <h2>{process.env.REACT_APP_SUBTITLE}</h2>
-    <Tasks />
-  </>
-);
-
-const Dashboard = () => (
-  <>
-    <h1>Dashboard</h1>
-  </>
-);
+import MDEditor from "@uiw/react-md-editor";
+import {
+  Routes,
+  Route,
+  Link,
+  useParams,
+  Navigate,
+  useNavigate,
+} from "react-router-dom";
+import slugify from "slugify";
+
+import { CommentList, AddComment } from "./Comments";
+import * as apiClient from "./apiClient";
+
+const App = () => {
+  return (
+    <>
+      <nav>
+        <Link to="/">Blog</Link> | <Link to="add-post">Write Post</Link>
+      </nav>
+      <main>
+        <Routes>
+          <Route path="/add-post" element={<WritePost />} />
+          <Route path="/" element={<Blog />} />
+          <Route path="/post/:id/:slug" element={<Post />} />
```

Comment:
where do you use the :/slug? I can not find any use of it

Reply:
are you trying to make it so you can load a post by its title?

Reply:
Yes, I was. I will submit a new PR and you can see my solution.